src/main/kotlin/no/nav/tilleggsstonader/sak/fagsak/Stønadstype.kt

```diff
@@ -1,5 +1,10 @@
 package no.nav.tilleggsstonader.sak.fagsak
 
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype as StønadstypeKontrakter
+
 enum class Stønadstype {
     BARNETILSYN,
 }
+fun StønadstypeKontrakter.tilInternType(): Stønadstype = when (this) {
```

Comment:
Vet ikke om vi faktisk trenger en intern og kontraktstype for `stønadstype`?

Reply:
Nei, vi burde egentligen kun bruke den i kontrakter.. Har bare ikke endret over sak til å bruke den som ble lagt til i konrakter 😬